.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+        required: true
+      - label: I have verified that the **issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php).**
+        required: true
+      - label: (optional but recommended) I have verified that the **issue is not fixed and is reproducible** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush).**
+        required: false
+      - label: I have **checked the [frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and **searched** the issue tracker for similar bug reports (including closed ones) **to avoid posting a duplicate.**
+        required: true
+      - label: This report is **not a support request or question**, both of which are better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/). The [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either.
+        required: true
+      - label: I have taken note of the **(ADVISORY)** & if relevant to issue or if asked by a **team member** will add the settings file and log(s) in the **Logs & settings file(s)** section at the bottom of the report. See [this wiki page](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings) if you're not sure where to find them.
+        required: false
+
+  - type: textarea
+    attributes:
+      label: Operating System & qBittorrent / Qt / libtorrent-rasterbar versions
+      description: Please specify if you are using 32-Bit or 64-Bit.
+      placeholder: |
+        qBittorrent: 4.3.7 x64
+        Qt: 5.15.2
+        libtorrent-rasterbar: 1.2.14
+        OS: Windows 10 Pro 21H1/2009 x64
+    validations:
+     required: true
+
+  - type: textarea
+    attributes:
+      label: What is the problem being experienced?
+      description: Please add "crash report" (if encountered) or give a clear and concise description of the bug/issue.
+    validations:
+     required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce.
+      description: Please walk us through it with a clear and concise description.
+      placeholder: |
+        1. First step
+        2. Second step
+        3. and so on...
+    validations:
+     required: false
+
+  - type: textarea
+    attributes:
+      label: What is the expected behavior?
+      description: Please give a clear and concise description of what you expected to happen.
+    validations:
+     required: true
+
+  - type: textarea
+    attributes:
+      label: Additional context.
+      description: Add screenshots etc. (Anything that will give us more context about the issue you are encountering!)
```

Comment:
Vocabulary should be constant I think, i.e. if there has already been used the word "problem", it should be kept being used.
```
A screenshots, screencasts, etc. 
(Anything that will provide more detailed description of the problem.)
```